js/web/lib/wasm/jsep/webgpu/ops/matmulnbits.ts

```diff
@@ -60,41 +59,27 @@ const validateInputs = (inputs: readonly TensorView[], attributes: MatMulNBitsAt
 export const createMatMulNBitsProgramInfo = (
   inputs: readonly TensorView[],
   attributes: MatMulNBitsAttributes,
-  maxComputeWorkgroupSizes: [number, number, number],
-  maxComputeWorkgroupStorageSize: number,
 ): ProgramInfo => {
   const inputShape = inputs[0].dims;
   const aRank = inputShape.length;
-  const nBlocksPerCol = Math.floor((attributes.k + attributes.blockSize - 1) / attributes.blockSize);
   const dimAOuter = inputShape[aRank - 2];
   const dimInner = attributes.k;
   const dimBOuter = attributes.n;
   const batchDims = inputShape.slice(0, aRank - 2);
   const batchSize = ShapeUtil.size(batchDims);
-  const blobSize = (attributes.blockSize / 8) * attributes.bits;
+  const blobSize = inputs[1].dims[2];
```

Comment:
The documentation probably is not correct. However, it says "Input B is a 2D constant Matrix" and in the box it says
"Input B is stored as uint8_t with shape: [N][n_blocks_per_col][blob_size] in which ..."
https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#commicrosoftmatmulnbits
Not sure if one should expect input B to be 3 dims.